.github/workflows/cmake.yml

```diff
@@ -2,6 +2,10 @@ name: Build and test
 
 on: [push, pull_request]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
I guess this can't be restricted to only pull requests?

Reply:
> I guess this can't be restricted to only pull requests?

:+1: We should probably build every merge on push just to make sure we can safely revert. If I understand the syntax, this should work:
`group: ${{ github.workflow }}-${{ github.event.pull_request.number}}`

Reply:
Thanks, I've changed it and checked that it cancels the PR workflows correctly